lms/static/js/spec/courseware/course_home_events_spec.js

```diff
@@ -21,19 +21,5 @@ define(['jquery', 'logger', 'js/courseware/course_home_events'], function($, Log
             $('.date-summary-link').click();
             expect(Logger.log).toHaveBeenCalledWith('edx.course.enrollment.upgrade.clicked', {location: 'sidebar'});
         });
-
-        it('sends an event when "Upgrade Now" is clicked from the upsell notification', function() {
-            $('.upgrade-banner-button').click();
-            expect(Logger.log).toHaveBeenCalledWith(
-                'edx.course.enrollment.upgrade.clicked', {location: 'notification'}
-            );
-        });
-
-        it('sends an event when "Learn More" is clicked from the upsell notification', function() {
-            $('.view-verified-info').click();
-            expect(Logger.log).toHaveBeenCalledWith(
-                'edx.course.home.learn_about_verified.clicked', {location: 'notification'}
```

Comment:
@dianakhuang: I think the new version of this is: 'edx.bi.dashboard.verified_info_link.clicked'.